test/runtests.jl

```diff
@@ -116,6 +116,18 @@ Base.iterate(cr::ConstRecord, state...) = iterate(_props(cr), state...)
         logger2 = getlogger("Pkg.Foo.Bar")
     end
 
+    @testset "DefaultRecord fields" begin
+        Memento.reset!()
+        root_io = IOBuffer()
+        push!(getlogger(), DefaultHandler(root_io, DefaultFormatter("[{threadid} - {pid} - {level}]: {msg}")))
+
+        msg = "Some message"
+        expected = Regex("\\[1 - \\d+ - warn\\]: $(msg)\\n")
+        Memento.warn(getlogger(), msg)
+        result = String(take!(root_io))
+        @test match(expected, result) !== nothing
+    end
```

Comment:
Should this go in test/records.jl?

Reply:
Yeah, I'd put most of this test in `test/records.jl` and if you wanna test the formatting you can always add a small test in `test/formatters.jl`. I don't think the logger or handler should care about this change.

Reply:
I see, makes sense.
I'll just add a small test in `test/records.jl`. Somehow missed that.